test/src/test/java/jenkins/security/ApiTokenPropertyTest.java

```diff
@@ -116,17 +123,38 @@ public void adminsShouldBeUnableToChangeTokensByDefault() throws Exception {
         assertEquals("Update token response is incorrect", 
                 Messages.ApiTokenProperty_ChangeToken_SuccessHidden(), "<div>" + res.getBody().asText() + "</div>");
     }
-    
+
+    @Test
+    public void postWithUsernameAndTokenInBasicAuthHeader() throws Exception {
+        FreeStyleProject p = j.createFreeStyleProject("bar");
+        j.jenkins.setSecurityRealm(j.createDummySecurityRealm());
+        User.get("foo");
+
+        WebClient wc = createClientForUser("foo");
+        WebRequest wr = new WebRequest(new URL(j.getURL(), "job/bar/build"), HttpMethod.POST);
+        wc.addCrumb(wr);
```

Comment:
This should be unnecessary since 2.96 or so.

Reply:
You are right, as always. Amended.